### PR TITLE
chore(concept):[#438] concept analysis and outcome documentation

### PR DESCRIPTION
**Pull request by mkanal**
concept analysis and outcome documentation

### CODE REVIEW

File: docs/concept/#438-investigation-dDTR-api-endpoints/AAS_Access_v3_Architecture.jpg
Comment (ds-jhartmann) [inline docs/concept/#438-investigation-dDTR-api-endpoints/AAS_Access_v3_Architecture.jpg:1]: Are we allowed to use this picture in our repo? Copyright belongs to Bosch

File: docs/concept/#438-investigation-dDTR-api-endpoints/#438-investigation-dDTR-api-endpoints.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+## Summary
+Extension of reference implementation of Digital Twin Registry (DTR) in Catena-X for scalable access management to Asset Administration Shells (AAS) and adhering Submodel server(s).
+
+## Assessment 
+
+- IRS uses the EDC for any request to the dDTR and Submodel Server 
+
+### Request Sequence (A) to API /lookup/shells
+
+**(1.) Data request from consumer EDC to provider EDC (A1)**
```
Comment (ds-jhartmann) [inline docs/concept/#438-investigation-dDTR-api-endpoints/#438-investigation-dDTR-api-endpoints.md:30]: If this is intended to be a headline, please use `#### (1.) Data request from consumer EDC to provider EDC (A1)` so the sections are displayed correctly

File: docs/concept/#438-investigation-dDTR-api-endpoints/AAS_Access_v3_Architecture.jpg
Comment (mkanal) [reply docs/concept/#438-investigation-dDTR-api-endpoints/AAS_Access_v3_Architecture.jpg:1]: This is public available and Copyright was not removed, so I think so.

File: docs/concept/#438-investigation-dDTR-api-endpoints/#438-investigation-dDTR-api-endpoints.md (hunk 1 above)
Comment (mkanal) [reply docs/concept/#438-investigation-dDTR-api-endpoints/#438-investigation-dDTR-api-endpoints.md:30]: No headline bold is enough.

Comment (mkanal) [reply docs/concept/#438-investigation-dDTR-api-endpoints/#438-investigation-dDTR-api-endpoints.md:30]: No headline indended.

**Review by ds-jhartmann — APPROVED** [review]
LGTM

**Pull request merged**